roles/vmrole/tasks/generatevm.yml

```diff
@@ -3,8 +3,8 @@
 # TODO: fetch template from API once loaded at deployment
 # TODO: Proper Linux and Windows templates are not yet ready, using the latest available
 - name: Fetch template
-  copy:
-    src: templates/vm-template-windows2012r2.yaml
+  template:
```

Comment:
@karmab , the `vm-template-windows2012r2` template is recently stored within the APB image but short-term plan is to take it from API only. Recently, it's exact copy of https://github.com/kubevirt/kubevirt/blob/master/cluster/vm-template-windows2012r2.yaml - so far the only one available.

The idea is to use templates produced by virt-team or v2v, so the OpenShift ones and not create our own.
Processing is done via `oc process`.

Reply:
so i opened https://github.com/kubevirt/kubevirt/pull/823 to host a second version of the windows template with a PVC_NAME as variable

Reply:
ok, in this case, i ve opened https://github.com/kubevirt/kubevirt/pull/823